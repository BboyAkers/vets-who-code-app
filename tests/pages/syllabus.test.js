import React from 'react'
import Syllabus from '../../src/pages/syllabus'

describe('<Syllabus />', () => {
  let wrapper

  beforeEach(() => wrapper = shallow(<Syllabus />))

  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
