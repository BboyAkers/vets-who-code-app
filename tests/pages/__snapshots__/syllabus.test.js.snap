// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Syllabus /> should render correctly 1`] = `
<div>
  <header
    className="inner-header overlay grey text-center slim-bg "
    style={
      Object {
        "backgroundImage": "url(image-mock)",
        "backgroundPositionY": "bottom",
      }
    }
  >
    <div
      className="overlay-01"
    />
    <div
      className="container"
    >
      <h2
        className="text-center text-uppercase"
      >
        Syllabus
      </h2>
      <div
        className="breadcrumb"
      >
        <GatsbyLink
          to="/"
        >
          Home
        </GatsbyLink>
        <span>
          /
        </span>
        <GatsbyLink
          className="page-active"
          to="/syllabus"
        >
          Syllabus
        </GatsbyLink>
      </div>
    </div>
  </header>
  <section
    className="faq section pad-regular bg-default"
    id="faq"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12"
        >
          <div
            className="faq-short-brief"
          >
            <h2>
              #VetsWhoCode Syllabus
            </h2>
            <p>
              Our curriculum is designed using the Agile Methodology with four-day sprints dedicated to the subject. We do it in this manner based on the practice of how veterans "Train How You Fight" in the military, so they should train how they would work in a real software company. The primary language we will focus on is JavaScript, due to its popularity in the job market, ease of transition between Computer Science topics and real world application, and the scope of industries we can introduce our veterans to, all while keeping language proficiency at the forefront of the program. This way we can introduce Data Visualization, Data Science, Cyber Security, and Internet of Things(IoT) to our veterans, while continuing to have our troops grow in the area of JavaScript.
            </p>
          </div>
        </div>
        <div
          className="col-md-12"
        >
          <div
            aria-multiselectable="true"
            className="panel-group faq_list"
            id="accordion"
            role="tablist"
          >
            <div
              className="panel panel-default active"
            >
              <div
                className="panel-heading"
                id="headingOne"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseOne"
                    aria-expanded="true"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseOne"
                    role="button"
                  >
                     
                    Week One: Set Up and Prep
                    <i
                      className="fa fa-minus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingOne"
                className="panel-collapse collapse in"
                id="collapseOne"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  During this week we focus on ensuring that our troops have the needed technologies set up and to gain exposure to our tools.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingTwo"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseTwo"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseTwo"
                    role="button"
                  >
                     
                    Week Two: The Web, Git and HTML
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingTwo"
                className="panel-collapse collapse"
                id="collapseTwo"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  This is when we start diving in on the basics, ensuring that you are comfortable in the terminal, and using version control, while teaching how the web works.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingThree"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseThree"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseThree"
                    role="button"
                  >
                     
                    Week 3: Basics of CSS
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingThree"
                className="panel-collapse collapse"
                id="collapseThree"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Here we start learning CSS with the focus on learning web design.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingFour"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseFour"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseFour"
                    role="button"
                  >
                     
                    Week 4: Advanced CSS
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingFour"
                className="panel-collapse collapse"
                id="collapseFour"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Here we focus on more advanced concepts such as Grid Layout as well as working with Foundation and Bootstrap.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingFive"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseFive"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseFive"
                    role="button"
                  >
                     
                    Week 5: JavaScript Basics
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingFive"
                className="panel-collapse collapse"
                id="collapseFive"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  This week we will start to learn the history and basics of Javascript and learn how to use it alone as well as in web design.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingSix"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseSix"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseSix"
                    role="button"
                  >
                     
                    Week 6: Computer Science With Javascript
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingSix"
                className="panel-collapse collapse"
                id="collapseSix"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  We continue our training in Javascript while also learning fundamentals in Computer Science. Recursion, Big O Notation, Sorting, Data Structures and Functional Programming.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingSeven"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseSeven"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseSeven"
                    role="button"
                  >
                     
                    Week 7: Computer Science With Javascript II: Deeper Dive
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingSeven"
                className="panel-collapse collapse"
                id="collapseSeven"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Here we continue to build Computer Science skills with javascript. Binary Trees and Breadth-First Search are just a few of the things we will dive into.
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingEight"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseEight"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseEight"
                    role="button"
                  >
                     
                    Week 8: Data Visualization
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingEight"
                className="panel-collapse collapse"
                id="collapseEight"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  More JavaScript and even more math as we enter the world of Canvas, D3.js, WebGL
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingNine"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseNine"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseNine"
                    role="button"
                  >
                     
                    Week 9: Node.js Express and Socket.io
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingNine"
                className="panel-collapse collapse"
                id="collapseNine"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Entering the world of Node.js and writing Javascript on the server
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingTen"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseTen"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseTen"
                    role="button"
                  >
                     
                    Node, Express and Socket.io II
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingTen"
                className="panel-collapse collapse"
                id="collapseTen"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Continuing to build with Node and learning Test Driven Development
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingE"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseE"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseE"
                    role="button"
                  >
                     
                    React
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingE"
                className="panel-collapse collapse"
                id="collapseE"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Start down the path or React, ES6 and Mongo DB
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingT"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseT"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseT"
                    role="button"
                  >
                     
                    React II
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingT"
                className="panel-collapse collapse"
                id="collapseT"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Continuing to develop in React
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingTh"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseTh"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseTh"
                    role="button"
                  >
                     
                    Interview Prep I
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingTh"
                className="panel-collapse collapse"
                id="collapseTh"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Start resume preparations as well as how to indentify potential job opportunities
                </div>
              </div>
            </div>
            <div
              className="panel panel-default"
            >
              <div
                className="panel-heading"
                id="headingFourteen"
                role="tab"
              >
                <h4
                  className="panel-title"
                >
                  <a
                    aria-controls="collapseFourteen"
                    aria-expanded="false"
                    className="collapsed"
                    data-parent="#accordion"
                    data-toggle="collapse"
                    href="#collapseFourteen"
                    role="button"
                  >
                     
                    Interview Prep II
                    <i
                      className="fa fa-plus accordions-derective-icon pull-right"
                    />
                  </a>
                </h4>
              </div>
              <div
                aria-labelledby="headingFourteen"
                className="panel-collapse collapse"
                id="collapseFourteen"
                role="tabpanel"
              >
                <div
                  className="panel-body"
                >
                  Starting the process of mock interviews and learning how to hack the technical interview.
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
